gapil/bapi/bapi.proto

```diff
@@ -420,6 +420,7 @@ message Expression {
     uint64 clone = 9;              // Clone ID
     uint64 create = 10;            // Create ID
     uint64 definition = 11;        // Definition ID
+    uint64 definition_usage = 43;  // Definition Usage ID
```

Comment:
Should this be `12` and all subsequent fields be incremented?

I know using `43` would be the right way to keep backward compatibility, but looking at the history of such proto definitions in GAPID/AGI, it looks like so far the approach has been to keep alphabetical order and change field numbers to match that.

I guess we might want to define on a project-wide rule for such proto updates.

Reply:
Proto fields shouldn't really be re-numbered. We've done it in the past, because we just didn't care.